{{cookiecutter.project_slug}}/webpack.config.js

```diff
@@ -27,9 +34,12 @@ module.exports = {
         test: /\.js$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
+        options: {
+          presets: [['@babel/preset-env', { modules: 'commonjs' }]],
+        },
```

Comment:
This should be moved in an external `babel.config.js` file at the root of the project so that other tools can benefit from it if they have to. Also I don't think we have to specify the `modules` option, the default `auto` is usually fine, isn't it?

Reply:
Also we should specify `useBuiltIns` to `usage` and set the `corejs` version to `3` to enable polyfills.